### PR TITLE
Update Linux Installation with Releases URL

### LINKED ISSUE #176: Shortcuts to installing Linux upgrades
There should be some quick link on https://docs.rancherdesktop.io/getting-started/installation/#linux for users just wanting to upgrade to the latest version.

For Windows and macOS the users just download the latest version directly from the Github release page, but for Linux they have to read through several blocks of text to find the relevant information.

### PR DESCRIPTION
**Pull request by sunilarjun**
Adding in a section to the installation guide for Linux users to quickly navigate to the releases page to download the application. This PR fixes #176 .

### CODE REVIEW

**Comment by adamkpickering** [comment]
This isn't correct - we don't want users to install Rancher Desktop from the .zip file on releases. While the .zip file does contain a working version of Rancher Desktop, it is not wrapped in an installer or any kind of packaging format. It is there for CI: when the release is published, it triggers a github workflow that uploads the .zip file to S3, and then triggers OBS to download that file and build the three supported package formats (deb, rpm and AppImage). It is these package formats that we want users to use when installing Rancher Desktop.

Basically, this change would mislead users. Can we please close it?

**Comment by sunilarjun** [comment]
Hi Adam, thanks for your review and clarification! I'll go ahead and close this one out.
